src/status_im2/common/standard_authentication/events.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im2.common.standard-authentication.events
+  (:require
+    [utils.re-frame :as rf]))
+
+(rf/reg-event-fx :standard-auth/on-biometric-success
+ (fn [{:keys [db]} [callback]]
+   (let [key-uid (get-in db [:profile/profile :key-uid])]
+     {:keychain/get-user-password [key-uid callback]})))
```

Comment:
If you want to follow the guidelines by the book, the effects map should only use `:fx` and `:db`.